ui/client/components/tips/ErrorhHeader.js

```diff
@@ -0,0 +1,8 @@
+import React from "react"
+
+export function ErrorHeader(props) {
```

Comment:
Do we need external file for this and for Node ErrorLink? If you use it only in Errors, maybe better way is creating two functional components in Errors?

Reply:
Yes, you're right, done